package.json

```diff
@@ -1,29 +1,51 @@
 {
   "name": "poeditorial",
-  "version": "1.3.2",
-  "description": "Minimal CLI tool for poeditor.com",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "version": "2.0.0",
+  "module": "src/index.ts",
   "bin": {
-    "poeditorial": "poeditorial.js"
+    "poeditorial": "dist/index.js"
   },
+  "files": [
+    "dist"
+  ],
+  "license": "MIT",
   "keywords": [
     "poeditor",
     "poeditor.com",
     "cli"
   ],
   "author": "Svetlozar Argirov <zarrro@gmail.com>",
-  "license": "MIT",
+  "type": "module",
   "repository": {
     "type": "git",
     "url": "https://github.com/zaro/poeditorial.git"
   },
+  "scripts": {
+    "build": "bun build src/index.ts --outdir=dist --target=node --packages external --splitting",
```

Comment:
typescript is already  devDependency. What is the point of uisng bun for building ?

I for example don't use bun, so it's one step more to building.

Reply:
If you publishing it from your local computer, then yes. I believe that github actions should publish the package automatically